src/applications/vaos/services/appointment/transformers.js

```diff
@@ -293,30 +264,44 @@ function setParticipant(appt) {
 
   switch (type) {
     case APPOINTMENT_TYPES.vaAppointment: {
-      if (!isVideoVisit(appt)) {
-        const participant = [];
+      const participant = [];
+      if (appt.clinicId) {
```

Comment:
Some video visits have clinics, so I changed this to just detect if there's a clinicId